CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
-## [UNRELEASED] neptune-client 0.16.6
+## [UNRELEASED] neptune-client 0.17.0
```

Comment:
Does it break compatibility? Btw. I would like to release PK's MR before increasing minor to 17.

Reply:
No it doesn't, but according to semver and [mycek](https://neptune-labs.slack.com/archives/C01H1A8DA3X/p1661849643485799) it should be minor.

Reply:
Let's increase PATCH in this case. These rules does not apply to version 0.

Reply:
ok